src/domain/event/eventForm/ValidationSchema.ts

```diff
@@ -29,4 +29,15 @@ export default Yup.object().shape({
   providerContactInfo: Yup.object().shape({
     email: Yup.string().email(VALIDATION_MESSAGE_KEYS.EMAIL),
   }),
+  image: Yup.string(),
+  photographer: Yup.string().when('image', {
```

Comment:
Maybe validateField method could be used to validate photographer and imageAltText fields after deleting image
https://jaredpalmer.com/formik/docs/guides/validation